pkgs/by-name/so/soapyplutosdr/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, darwin
+, libad9361
+, libiio
+, libusb1
+, soapysdr
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "soapyuhd";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "pothosware";
+    repo = "SoapyPlutoSDR";
+    rev = "soapy-plutosdr-${finalAttrs.version}";
+    hash = "sha256-uXKvv/QRbYknqsLGlPFxSH7KLh0CucLjq4XEFFcieWw=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [
+    libad9361
+    libiio
+    libusb1
+    soapysdr
+  ]
+  ++ lib.optionals stdenv.isDarwin [
```

```suggestion
  ++ lib.optionals stdenv.hostPlatform.isDarwin [
```
Though I think you can omit the first two lines below.